Tests/DatadogTests/Datadog/Mocks/DatadogInternal/DatadogContextMock.swift

```diff
@@ -60,6 +62,34 @@ extension DatadogContext: AnyMockable {
             featuresAttributes: featuresAttributes
         )
     }
+
+    static func mockRandom() -> DatadogContext {
+        .init(
+            site: .mockRandom(),
+            clientToken: .mockRandom(),
+            service: .mockRandom(),
+            env: .mockRandom(),
+            version: .mockRandom(),
+            variant: .mockRandom(),
+            source: .mockRandom(),
+            sdkVersion: .mockRandom(),
+            ciAppOrigin: .mockRandom(),
+            serverTimeOffset: .zero,
```

Comment:
`.zero` might be very unfortunate value for fuzzy testing. Could we make it random 🙌? I think there is `.mockRandomInThePast()` defined on `TimeInterval` - IMO better this than `0`.